docs/programming-oak.md

```diff
@@ -240,7 +239,7 @@ previous section (which would typically be published by the ISV providing the
 Oak Application).
 
 The client connects to the gRPC service, and sends (Node-specific) gRPC requests
-to it, over a channel that has end-to-end encryption into the enclave:
+to it, over a channel that has end-to-end encryption:
```

Comment:
Perhaps "into the runtime instance"? though I guess that's obvious

Reply:
Done.